### ISSUE TITLE
NullReferenceException exception when RemoveOwnership()

### ISSUE THREAD

**Issue by ooonush** [FirstGearGames/FishNet#408]
# Important
If General, Description, and Replication are not completed the issue will be closed immediately.

**General**
Unity version: 2022.3.0
Fish-Networking version: 3.7.10

**Description**

When stopping the play mod on the server while calling RemoveOwnership(), it gives a NullReferenceException error for NetworkObservers.

```csharp
public override void OnDespawnServer(NetworkConnection connection)
{
    base.OnDespawnServer(connection);

    if (Owner == connection)
    {
        RemoveOwnership();
    }
}
```

![image](https://github.com/FirstGearGames/FishNet/assets/72870405/6f7101a6-ea14-426d-8c35-3576825ca927)

**Comment by FirstGearGames** [comment]
Harmless but resolved in 3.7.11

**Comment by ooonush** [comment]
When loading another scene, a similar error occurs:
![image](https://github.com/FirstGearGames/FishNet/assets/72870405/778c8fc4-7857-476f-bba0-5facd9462c3b)

```csharp
public override void OnDespawnServer(NetworkConnection connection)
{
    base.OnDespawnServer(connection);

    if (Owner == connection)
    {
        RemoveOwnership();
    }
}
```

```csharp
var sceneLoadData = new SceneLoadData(SceneName.CapturePointPostMatchScene)
{
    ReplaceScenes = ReplaceOption.All
};
sceneLoadData.PreferredActiveScene = sceneLoadData.SceneLookupDatas[0];

SceneManager.LoadGlobalScenes(sceneLoadData);
```
